scripts/testnet/start.sh

```diff
@@ -11,6 +11,12 @@ source "$ELRONDTESTNETSCRIPTSDIR/include/build.sh"
 source "$ELRONDTESTNETSCRIPTSDIR/include/nodes.sh"
 source "$ELRONDTESTNETSCRIPTSDIR/include/tools.sh"
 
+# Load local overrides, .gitignored
+LOCAL_OVERRIDES="$ELRONDTESTNETSCRIPTSDIR/local.sh"
+if [ -f "$LOCAL_OVERRIDES" ]; then
+  source "$ELRONDTESTNETSCRIPTSDIR/local.sh"
+fi
+
```

Comment:
`variables.sh` already loads `local.sh`, did you need to override funtions in the `include/*.sh` files?

Reply:
removed